core/block_render_svg_vertical.js

```diff
@@ -651,6 +658,13 @@ Blockly.BlockSvg.prototype.render = function(opt_bubble) {
 
   this.renderClassify_();
 
+  // Position the Scratch Block Comment Icon at the end of the block
+  if (scratchCommentIcon) {
+    var iconX = this.RTL ? -inputRows.rightEdge : inputRows.rightEdge;
```

Comment:
Did you check that this positions it correctly in RTL?

Reply:
RTL wasn't working for me in the vertical_playground so I wasn't able to test this, I still get this error on develop:

```
flyout_base.js:609 Uncaught TypeError: Cannot set property 'length' of undefined
    at Blockly.VerticalFlyout.Blockly.Flyout.recordCategoryScrollPositions_ (flyout_base.js:609)
    at Blockly.VerticalFlyout.getMetrics_ (flyout_vertical.js:211)
    at Blockly.WorkspaceSvg.getWidth (workspace_svg.js:830)
    at Object.Blockly.Xml.blockToDomWithXY (xml.js:86)
    at new Blockly.Events.Create (block_events.js:226)
    at Object.Blockly.Xml.domToBlock (xml.js:586)
    at Blockly.VerticalFlyout.Blockly.Flyout.show (flyout_base.js:506)
    at Blockly.Toolbox.showAll_ (toolbox.js:209)
    at Blockly.Toolbox.populate_ (toolbox.js:186)
    at Blockly.Toolbox.init (toolbox.js:135)
```

Reply:
https://github.com/LLK/scratch-blocks/issues/1548